f5/bigip/resource.py

```diff
@@ -161,6 +161,11 @@ class UnsupportedOperation(F5SDKError):
     pass
 
 
+class InvalidStatsJsonReturned(KeyError):
+    """Returned stats JSON should always contain 'entries' key"""
+    pass
+
```

Comment:
I dont think this is always the case. Here's an example of exemplary engineering

```
{
  "kind": "tm:sys:service:servicecollectionstats",
  "selfLink": "https://localhost/mgmt/tm/sys/service/stats?ver=11.6.0",
  "apiRawValues": {
    "apiAnonymous": "acctd        down, Not provisioned\naced         down, Not provisioned\nadmcloud     down, Not provisioned\nadmd         down, Not provisioned\nalertd..."
  }
}
```